### ISSUE TITLE
'renderer' argument is required for future Django versions

### ISSUE THREAD

**Issue by cachitas** [fabiocaccamo/django-admin-interface#34]
Using Django 2.1b1, the following error arises when accessing a theme change view.

```
TypeError at /admin/admin_interface/theme/1/change/
render() got an unexpected keyword argument 'renderer'
```

As noted in 2.1 release notes: 
> Support for Widget.render() methods without the renderer argument is removed.

**Comment by fabiocaccamo** [comment]
Could you write a test case for this, please?

**Comment by fabiocaccamo** [comment]
I understand, django-colorfield should be rewritten using a more modern color picker and with a test suite.

**Comment by zvolsky** [comment]
I have made PR for django-colorfield
https://github.com/jaredly/django-colorfield/pull/41.
Or django-colorfield can be dowloaded from
git@github.com:zvolsky/django-colorfield.git.
Or add manually new parameter(s): def render(....., renderer=None, **_kwargs)

**Comment by fabiocaccamo** [comment]
@zvolsky thanks, I will release a `django-colorfield` minor update today.

**Comment by fabiocaccamo** [comment]
@zvolsky released `0.1.15` version of `django-colorfield`

**Comment by cachitas** [comment]
Just tested again and everything looks fine. tyvm
